documentation/source/library_reference.rst

```diff
@@ -30,6 +30,7 @@ Writing and Generating tests
     :members:
     :member-order: bysource
 
+.. autofunction:: cocotb.decorators.RunningCoroutine.log
```

Comment:
This isn't a function, it's a property.

I'm not even sure it's worth documenting - I can't think of any scenario where you both actually have access to this object, and want to use it. It seems to only be for internal use.

Reply:
Removed.